packages/office-ui-fabric-react/src/components/Stack/examples/Stack.Horizontal.Configure.Example.tsx

```diff
@@ -33,6 +33,36 @@ export interface IExampleOptions {
   emptyChildren: string[];
 }
 
+// Alignment options
+const horizontalAlignmentOptions: IDropdownOption[] = [
+  { key: 'start', text: 'Left' },
+  { key: 'center', text: 'Center' },
+  { key: 'end', text: 'Right' },
+  { key: 'space-around', text: 'Space around' },
+  { key: 'space-between', text: 'Space between' },
+  { key: 'space-evenly', text: 'Space evenly' },
+];
+const verticalAlignmentOptions: IDropdownOption[] = [
+  { key: 'start', text: 'Top' },
+  { key: 'center', text: 'Center' },
+  { key: 'end', text: 'Bottom' },
+];
+
+// Tokens definition
+const sectionStackTokens: IStackTokens = { childrenGap: 10 };
+const configureStackTokens: IStackTokens = { childrenGap: 20 };
+
+const shadowItemCheckboxStyles: Partial<ICheckboxStyles> = { root: { marginRight: 10 } };
+const wrapItemCheckboxStyles: Partial<ICheckboxStyles> = { root: { marginBottom: 10 } };
+
+const range = (start: number, end: number): number[] => {
```

Comment:
Can you add this to `packages/utilities/src/array.ts` and use that everywhere?

Reply:
This helper creates an array of incrementing numbers. It probably should be named `createNumberList` or something, but I don't think we would need a helper like that in array utilities; possibly test-utilities though.

Reply:
`range` is the name used for this in [lodash](https://docs-lodash.com/v4/range/) and python, and likely elsewhere too. So I think we should keep the name.